packages/docs/docs/guides/transactions.mdx

```diff
@@ -43,6 +41,16 @@ Example below clears inputs and enables all disabled components back:
     }, [state])
 ```
 
+**Live example**
+
+App will send 0.5 ether to a test account.
```

Comment:
Why so much? It's free ether but still in my experience testnet ether is not always easy to get when faucets are dry.
Let's make it 1 Gwei.